kratky_rg_izero.py

```diff
@@ -153,9 +178,6 @@ def plot_rg_io(data_dir, outfile, colors):
         l, = ax.plot(df['s_norm'], df['j_exp_norm'],
                      linestyle="", marker="o", color=color)
```

Comment:
fwiw, you are using double-quote on this line, but single-quote in most other places. Single quotes ftw.